libs/sdk-core/src/models.rs

```diff
@@ -1098,6 +1109,23 @@ pub struct Channel {
     ///
     /// This may be empty for older closed channels, if it was not possible to retrieve the closing txid.
     pub closing_txid: Option<String>,
+
+    pub htlc: Vec<Htlc>,
+}
+
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub struct Htlc {
+    pub expiry: u32,
+    pub payment_hash: Vec<u8>,
+}
+
+impl Htlc {
+    pub fn from(expiry: u32, payment_hash: Vec<u8>) -> Self {
+        Htlc {
+            expiry,
+            payment_hash,
+        }
+    }
```

Comment:
I don't think this is used, could be removed

Reply:
I did not remove it and instead use it when listing channel peers. Not sure which approach is preferred.